src/Microsoft.Fast.Components.FluentUI/Components/DesignSystemProvider/FluentDesignSystemProvider.razor

```diff
@@ -64,6 +64,7 @@
             neutral-stroke-active-delta="@NeutralStrokeActiveDelta"
             neutral-stroke-focus-delta="@NeutralStrokeFocusDelta"
             base-layer-luminance="@BaseLayerLuminance?.ToString(System.Globalization.CultureInfo.InvariantCulture)"
+            style="@Style"
```

Comment:
Why do you want/would you need this? The component itself has no visual representation at all.

Reply:
For a component with the style="height:100%" attribute in HTML, it only works when all of its parent elements have the style="height:100%" set.

Reply:
Ok, makes sense